media/js/firefox/hello/index.js

```diff
@@ -2,7 +2,7 @@
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-;(function(w, $, Modernizr) {
+;(function(Mozilla, w, $, Modernizr) {
```

Comment:
After a closer look here, I think we can replace `Modernizr` with `Mozilla.SVGAnimCheck.supportsCSSAnimations`. The use of `Modernizr` here was one of caution anyway (only for mobile/small devices that don't support CSS animations) - better to lighten the JS load and use your version?